### ISSUE TITLE
Deserialize MesosCluster based on minimesosFile

### ISSUE THREAD

**Issue by frankscholten** [ContainerSolutions/minimesos#365]
Currently the `minimesosFIle` is only used when running `minimesos init`. We should however change the code so the `MesosCluster` object is deserialized based on the `minimesosFile`.

As suggested by @alex-glv let's move the `minimesosFile` to the cluster specific `.minimesos` dir after `up`. Any subsequent cli command will use that file to retrieve a cluster.

**Comment by alex-glv** [comment]
I liked @lguminski's idea to create an intermediate cluster state file, this will be low-level cluster representation based on the configfile that was used to start the cluster and also id's of the containers that that compose the running cluster

**Comment by frankscholten** [comment]
Ok, so you mean expanding the `minimesos.cluster` file we now have

**Comment by sadovnikov** [comment]
this one depends on #327

**Comment by frankscholten** [comment]
Duplicate of #327 as this implemented in that issue.
